### ISSUE TITLE
ssu-device-enable and ssu-device-disable scripts

### ISSUE THREAD

**Issue by richardsj** [lamassu/lamassu-scripts#51]
In the situation a machine is in an unknown or insecure state (e.g. has left a geofencing region, isn't reachable etc) - it's advantageous to be able to disable the machine quickly without unpairing it (for obvious reasons).  This can even be part of an overnight cron job for locations that close between certain hours.

Arguments: blank (displays status) | status | all | `<device fingerprint>` | `<device name>`

**Comment by naconner** [comment]
Good idea, Joe. FYI, we're migrating away from scripts in the next-gen, so it'll be fully admin-based. Worth considering for iterations of that.
